### ISSUE TITLE
[RTL] Comments

### ISSUE THREAD

**Issue by Siemienik** [handsontable/handsontable#8822]
### Description

For RTL, the comment icon should be displayed on the left and the comment dialog should open from right to left.

### How to reproduce:

1. Use this [configuration](https://github.com/Siemienik/handsontable-storybooks/blob/610f03de38a979a1c65d71b08897786140699a91/vanilla/configurations.js#L747-L764):

```js
{
    data: [
        ['', 'Tesla', 'Nissan', 'Toyota', 'Honda', 'Mazda', 'Ford'],
        ['2017', 10, 11, 12, 13, 15, 16],
        ['2018', 10, 11, 12, 13, 15, 16],
        ['2019', 10, 11, 12, 13, 15, 16],
    ],
    rowHeaders: true,
    colHeaders: true,
    contextMenu: true,
    comments: true,
    cell: [
        { row: 1, col: 1, comment: { value: 'Some comment' } },
        { row: 2, col: 2, comment: { value: 'Comment 200x50 px', style: { width: 200, height: 50 } } }
    ],
    height: 'auto',
    licenseKey: 'non-commercial-and-evaluation'
}
```

2. result:
![image](https://user-images.githubusercontent.com/2893099/135889776-b71fc752-a289-417e-933a-8b1b5af7c282.png)

**Comment by aninde** [comment]
Fixed in the v12.0.0 https://jsfiddle.net/jz6amgx3/1/

**Comment by AMBudnik** [comment]
Closed with https://github.com/handsontable/handsontable/issues/8760 (RTL Support), published in Handsontable v12.

For more information please check out our Release Notes - https://handsontable.com/docs/release-notes/#_12-0-0
